config/initializers/ratonvirus.rb

```diff
@@ -1,9 +1,9 @@
 Ratonvirus.configure do |config|
-  if Rails.env.production?
-    config.scanner = :resty
-    config.storage = :active_storage
-  else
+  if Rails.env.test?
```

Comment:
Discussed last week re Resty being available to test in Test env, as testing 'eicar' is different code.  This does not appear to have changed.

Reply:
this is the test environment for automated tests run in rails, nothing to do with the completely separate space called 'test'

Reply:
My mistake no name change required.  So we can now run this is test space, which is great.

Reply:
The server is currently open to the public, by design

The server is only being used to receive and file and send backs a message on whether that particular file is infected or not.  It does not change any of the existing mechanisms, so files remain in their respective spaces